src/init.jl

```diff
@@ -22,6 +17,19 @@ end
 
 ## target initialization
 
+# figure out the supported targets by looking at initialization routines
+# TODO: figure out the name of the native target
```

Comment:
Yeah `llvm-config` get's it from a build header... The only way I know to do it proberly is to parse llvm-config or the exported CMake files.

Reply:
Yeah, it's annoying they bake so much in headers and build flags, which we can't use (since it doesn't guarantee it's the one for the `libLLVM` we've picked up)